src/IconView.vala

```diff
@@ -103,6 +104,26 @@ public class IconView : Gtk.Box {
         source_view.add_css_class (Granite.STYLE_CLASS_CARD);
         source_view.add_css_class (Granite.STYLE_CLASS_ROUNDED);
 
+        copy_button = new Gtk.Button.with_label (_("Copy")) {
+            valign = START,
+            halign = END,
+            visible = false,
+            margin_top = 8,
+            margin_end = 8
+        };
+        copy_button.add_css_class (Granite.STYLE_CLASS_SMALL_LABEL);
+        copy_button.add_css_class ("copy-button");
+
+        var copy_visible_controller = new Gtk.EventControllerMotion ();
+        copy_visible_controller.bind_property ("contains-pointer", copy_button, "visible",
+                                                BindingFlags.DEFAULT | BindingFlags.SYNC_CREATE);
```

Comment:
Only showing on hover is problematic because it means this feature is unavailable on touch. It looks like on the docs site the button also shows if the code snippet contains focus. So we could do that or just always show it

This is kind of a funky indentation style. If you don't include the `BindingFlags` namespace you can have all the args on one line like:
```suggestion
        copy_visible_controller.bind_property (
            "contains-pointer", copy_button, "visible", DEFAULT | SYNC_CREATE
        );
```